### DISCUSSION TITLE
Can I draw an image on a given position inside a window?

### DISCUSSION THREAD

**Discussion by dani-lp** [ocornut/imgui#3965] [Old/Archived discussions]
In case it's possible, can I draw an image over another image?

**Comment by rokups** [comment]
Position a cursor to desired position before drawing an image. When you render items one on top of the other - last `Image()` call would be drawn on top.
```cpp
ImGui::SetCursorPos(x, y);
// Or ImGui::SetCursorScreenPos(x, y);
ImGui::Image(...)
```
